src/DependencyEmitter/UsesDependencyEmitter.php

```diff
@@ -34,4 +37,33 @@ public function applyDependencies(AstMap $astMap, Result $dependencyResult): voi
             }
         }
     }
+
+    protected function isNamespace(AstDependency $dependency, AstMap $astMap): bool
+    {
+        $dependencyFQDN = $dependency->getTokenName()->toString();
+
+        $functionReferences = array_merge(...array_values(array_map(
```

Comment:
Is there a better way to get all function references from `AstMap`? There is a list of function references in `AstMap` but no getter exposes it.

Reply:
If you need at and there is no nice way to get it, just add the getter. :)